Tester/Runner/CliTester.php

```diff
@@ -184,9 +184,16 @@ private function createRunner()
 		$runner->stopOnFail = $this->options['--stop-on-fail'];
 
 		if ($this->options['-o'] !== 'none') {
-			$runner->outputHandlers[] = $this->options['-o'] === 'tap'
-				? new Output\TapPrinter($runner)
-				: new Output\ConsolePrinter($runner, $this->options['-s']);
+			switch ($this->options['-o']) {
+				case 'tap':
+					$runner->outputHandlers[] = new Output\TapPrinter($runner);
+					break;
+				case 'junit':
+					$runner->outputHandlers[] = new Output\JUnitWriter($runner);
```

Comment:
Yeah, now it does. I used Writer because original purpose of this class was writing xml